WhisperAPI/WhisperAPI/Controllers/ContextController.cs

```diff
@@ -21,20 +21,34 @@ public ContextController(IContexts contexts)
 
         public override void OnActionExecuting(ActionExecutingContext actionExecutingContext)
         {
-            var searchQuery = (SearchQuery)actionExecutingContext.ActionArguments["searchQuery"];
-
-            log4net.ThreadContext.Properties["requestId"] = Guid.NewGuid();
             if (!this.ModelState.IsValid)
             {
                 actionExecutingContext.Result = this.BadRequest(this.ModelState);
-                Log.Error($"Search query:\r\n{JsonConvert.SerializeObject(searchQuery, Formatting.Indented)}");
```

Comment:
Is there a way we could keep this Log.Error ? Maybe we could do the TryGetValue first so we could have the searchQuery (if any).

Reply:
Why would we want to log bad user input data ? I understand when it is related to server data but model state is http 400, it is not a error on our part.